cmd/helm/root.go

```diff
@@ -95,6 +95,26 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 		Short:        "The Helm package manager for Kubernetes.",
 		Long:         globalUsage,
 		SilenceUsage: true,
+		PersistentPreRun: func(cmd *cobra.Command, _ []string) {
+			cpuprof, err := cmd.Flags().GetString("cpuprofile")
```

Comment:
Would it be better to expose profiling as env vars? e.g. 

```
HELM_PPROF=cpu=/path/to/cpu.prof,mem=/path/to/mem.prof
```

The rationale is this seems a little less intrusive than cli flags. Thoughts?

Reply:
Is there prior art to CLIs adding pprof support too?

Reply:
Using env vars makes sense. I'll make the change

> Is there prior art to CLIs adding pprof support too?

Besides https://github.com/replicatedhq/troubleshoot, which is a project I contribute to, I have not come across any other. I know web server implementations such as Gorilla Mux and Negroni expose debug info using https://pkg.go.dev/net/http/pprof.

Reply:
For what its worth, golang recommends this usage https://pkg.go.dev/runtime/pprof